edit-post/README.md

```diff
@@ -7,7 +7,9 @@ Refer to [the plugins module documentation](../plugins/) for more information.
 ## Plugin Components
 
 The following components can be used with the `registerPlugin` ([see documentation](../plugins)) API.
-They can be found in the global variable `wp.editPost` when defining `wp-edit-post` as a script dependency.
+They can be found in the global variable `wp.editPost` when defining `wp-edit-post` as a script dependency. 
+
+**Note:** Plugin scripts should be enqueued to footer by setting `in_footer` option to true ( [see wp_enqueue_script documentation](https://developer.wordpress.org/reference/functions/wp_enqueue_script/) ).
```

Comment:
@gziolo, Is `in_footer` the best way to handle this? I burned some time chasing down "Target container is not a DOM element" error that appeared whenever the wp-edit-post script was included. Putting it in the footer resolved the issue. 
![not in footer error](https://user-images.githubusercontent.com/6209518/38145246-39b80174-33fd-11e8-959a-044553e636be.PNG)

Reply:
Not sure to be honest. I will investigate it later this week. Ideally, it should be enough to set edit-post as a dependency of your plugin.

Reply:
The content of my script seemed to be totally irrelevant. My best guess is that edit-post has a dependency that was only there in time if I put it in the footer.

Reply:
we may be injecting some scripts on action hooks without regard to dependencies (ie without wp_enqueue_script)? that could cause this.

Reply:
It has been fixed, you just need to make sure you provide all dependencies when calling `wp_enqueue_script`.